torchvision/utils.py

```diff
@@ -382,9 +391,16 @@ def draw_keypoints(
     return torch.from_numpy(np.array(img_to_draw)).permute(2, 0, 1).to(dtype=torch.uint8)
 
 
-def _generate_color_palette(num_masks: int):
+def _generate_color_palette(num_masks: int, return_tensor: bool = True):
     palette = torch.tensor([2 ** 25 - 1, 2 ** 15 - 1, 2 ** 21 - 1])
-    return [tuple((i * palette) % 255) for i in range(num_masks)]
+    clrs = []
+    for i in range(num_masks):
+        clr = (palette * i) % 255
+        if return_tensor:
+            clrs.append(tuple(clr))
+        else:
+            clrs.append(tuple((clr.tolist())))
```

Comment:
Nit: this can still be simplified up as discussed above.

Reply:
I don't see this `return_tensor` being used anywhere: do we really need it?

Reply:
yeah it can be :( Did this in a hurry. 
I guess best way is
```
if not return_tensor:
    clr.tolist()
clrs.append(tuple(clr))
return clrs
```